### ISSUE TITLE
Roboto-min next hinted build

### ISSUE THREAD

**Issue by dberlow** [TypeNetwork/Roboto#45]
After #42 and #44 are corrected, and we get a better idea of what’s needed for #43, there will be another build.

While this is going on, Mike is making sure the optimizations to crossbars and other details are performed on small caps and other target glyphs, and researching possible file size optimization. 

Marc has also requested a build chain for this project of similar quality to the original Roboto repo where the variable font can be built with a single command. I am looking into this and will post a new issue on it shortly. With that, I hope the next build can illustrate what can and can’t be the same for a variable font, that works from a single command for a family of fonts. 

Thanks,
David Berlow

**Comment by sannorozco** [comment]
Updat6ed new build, with fixes #42 #43 #44

**Comment by davelab6** [comment]
Sounds great, thanks DB and thanks Santiago for smashing through those 3 issues

**Comment by dberlow** [comment]
Mike, 

https://github.com/TypeNetwork/Roboto/blob/master/fonts/Roboto-min-VF.ttf

..has the new unhinted version we are looking to merge your hints into. 

Let me know if there's any issues.

**Comment by mikedug** [comment]
hints are now merged, VTT sources and ship font updated.

**Comment by m4rc1e** [comment]
Can we start to think about how we're going to do this hint transfer automatically? we cannot have Mike manually transferring the hints every time we make an update. 

DaMa has [vttLib](https://github.com/daltonmaag/vttLib) which we may want to use.

**Comment by dberlow** [comment]
Marc,

Thanks for your concern. We have thinking about automation every time we do anything on this project since the issue was first raised.

Thanks for the link to the vtt lib model. I’m traveling today to New York to visit Google and have a conference call later on this topic with Santiago and Mike.

Cheers
David

> On Feb 12, 2020, at 7:46 AM, Marc Foley <notifications@github.com> wrote:
> 

﻿
Can we start to think about how we're going to do this hint transfer automatically? we cannot have Mike manually transferring the hints every time we make an update.

DaMa has vttLib which we may want to use.

**Comment by sannorozco** [comment]
Hallo Marc,

Added atomated ttx remove MVAR each build to sh build script.
